index.html

```diff
@@ -271,5 +271,59 @@ <h2 class="categorie-title">
     </ul>
 </section>
 </main>
+<footer class="footer-container" id="footer-container-id">
+  <div class="footer-form">
+    <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+    <form action="https://formspree.io/f/xzbovlzk" method="post">
+      <ul>
+        <li>
+          <input type="text" name="Name" placeholder="Full name">
+        </li>
+        <li>
+          <input type="email" name="Email" placeholder="Email address">
+        </li>
+        <li>
+          <textarea name="Message" id="msg" cols="30" rows="10">Enter text here</textarea>
+        </li>
+        <li class="submit-button">
+          <button type="submit">
+              Get in touch
+          </button><br>
+          <small></small>
+      </li>
```

Comment:
- [ ] Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template, Kindly add the `hover` effect to your `Get in touch` button look :arrow_down: 

```
button:hover {
   box-shadow: 0px 8px 16px rgba(54, 179, 127, 0.24);
}
```